dist/js/materialize.js

```diff
@@ -220,8 +217,15 @@ jQuery.extend( jQuery.easing,
-    if (typeof define === 'function' && define.amd) {
-        define(['jquery', 'hammerjs'], factory);
+    if (typeof define === 'function' && define.amd == false) {
+        // hammerjs will already be publicly available
+        define(['jquery'], function($, Hammer) {
```

Comment:
and also by the way your suggestion is amazing but like i said it is not possible that's what the guys that created Require did and it's the AMD spec and i didn't wrote it so i don't know why they don't allow you to do such a thing, anyways i think now you know why i rely on hammer being publicly available vs. amd version of it, and note that you don't have to include hammerjs file in require config all you need is the materialize file because it makes no sense to have the same code twice which is the hammer library, this solution works and tested but what i haven't tested is what i talked about so there is no problem in using it i guess, only if you have any problems just include materialize from the bin directory and it will work fine (of course using require not the normal way)